.github/workflows/base.yaml

```diff
@@ -18,27 +18,25 @@ concurrency:
 
 jobs:
   build:
+    name: Build package & run tests
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1 # Use shallow clone for faster checkout
 
       - name: Check broken links
         uses: JustinBeckwith/linkinator-action@v1
         with:
           paths: "**/*.md"
 
-      - name: Get Flutter version by FVM
-        uses: kuhnroyal/flutter-fvm-config-action@v2
-        id: fvm-config-action
-        with:
-          path: '.fvmrc'
-          flavor: 'stable'
-
-      - uses: subosito/flutter-action@v2
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
         with:
-          flutter-version: ${{ steps.fvm-config-action.outputs.FLUTTER_VERSION }}
-          channel: ${{ steps.fvm-config-action.outputs.FLUTTER_CHANNEL }}
+          channel: "stable"
+          architecture: x64
```

Comment:
The use of FVM is for know what version is used on development.
If stable version will upgrade and the package is not upgraded, couldn't create new PR and use pipelines.

With this change, we don't have control of Flutter version that package is built